modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceInventory.testcase

```diff
@@ -401,6 +512,237 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12326. If the SKU/UOM combination is set, the admin deletes a placed order, the inventory quantities from the SKU/UOM combination can be restocked correctly"
+	@priority = 5
+	test CanRestockInventorySKUUOMByDeletingTheOrder {
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false${line.separator}feature.flag.COMMERCE-11287=true";
+		property portal.acceptance = "true";
+
+		task ("Given the buyer adds product ABS Sensor to the cart as quantity 2") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/p/abs-sensor");
+
+			FrontStore.addInitializerProductToCartFromProductDetails(
+				productName = "ABS Sensor",
+				productQuantity = 2);
+		}
+
+		task ("And the admin adds one UOM to SKU MIN93015(for product ABS Sensor)") {
+			CommerceJSONSKUsAPI._addCommerceUOM(
+				basePrice = 10,
+				productSKU = "MIN93015",
+				uomKey = "uomKey1",
+				uomName = "uomName1");
+		}
+
+		task ("When the buyer completes the checkout") {
+			CommerceCheckout.gotoCheckoutViaMiniCart();
+
+			CommerceCheckout.checkout(
+				newAddressAddress = "Test Address",
+				newAddressCity = "Test City",
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				shippingMethod = "Standard");
+		}
+
+		task ("And the admin searches for MIN93015 on the inventory admin page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "test test");
+
+			CommerceInventory.openInventoryAdmin();
+
+			CommerceNavigator.searchEntry(entryName = "MIN93015");
+		}
+
+		task ("Then the quantities for UOM1 On Hand:140, Available:138, On Order:2, Incoming:0") {
```

Comment:
Then the quantities for UOM1 can be viewed as On Hand:140, Available:138, On Order:2, Incoming:0